src/OrderManager/Order/Listing.php

```diff
@@ -75,16 +75,15 @@ public function getQueryBuilder(): DoctrineQueryBuilder
                 $queryBuilder->groupBy('OrderId');
             }
 
-            // filter order state
-            if (!is_null($this->getOrderState())) {
-                $orderStates = [];
-                foreach ((array)$this->getOrderState() as $orderState) {
-                    $orderStates[] = $queryBuilder->expr()->literal($orderState);
-                }
-
-                $queryBuilder->andWhere('order.orderState IN('. implode(',', $orderStates) .')');
+            // filter order state default order state is committed
+            $orderStates = [];
+            foreach ((array)$this->getOrderState() as $orderState) {
```

Comment:
```suggestion
            foreach ((array)$this->getOrderState() as $orderState) {
```
https://github.com/pimcore/ecommerce-framework-bundle/blob/e6ca0e41fc37f22ffafec47660ea3faca975d66b/src/OrderManager/AbstractOrderList.php#L68

it seems that the type is a `string`, weird, how it can be casted to array. 
At least, would expect an `explode` or similar functions, or it could be that is wrongly return `string`, can you double-check?

Reply:
This does not make sense at all tbh. How can it have more than one orderState anyways?